globals/utilities.php

```diff
@@ -237,6 +239,36 @@ function neve_get_svg_allowed_tags() {
 	);
 }
 
+/**
+ * Add extra allowed HTML tags to wp_kses()
+ *
+ * @return array Extra tags needed by the Jump to Page feature markup that are stripped by wp_kses() by default.
+ */
+function jump_to_page_extra_tags() {
```

Comment:
This should not be necessary since most of the content that it escapes is hardcoded, not something added by the user or retrieved from the DB. We can escape each of the form dynamic vars in particular and should be fine. 

Also as a sidenote, generally that kind of functions needs to be prefixed.